teos-common/src/test_utils.rs

```diff
@@ -48,7 +47,15 @@ pub fn generate_random_appointment(dispute_txid: Option<&Txid>) -> Appointment {
     };
 
     let tx_bytes = Vec::from_hex(TX_HEX).unwrap();
-    let penalty_tx = consensus::deserialize(&tx_bytes).unwrap();
+    let mut penalty_tx: Transaction = consensus::deserialize(&tx_bytes).unwrap();
+
+    // Append a random-sized OP_RETURN to make each transcation random in size.
```

Comment:
This should be OK as long as we never use this transaction for verification logic (nor broadcast over a real network), given the signature will never verify.

The inputs may not even be available now that I think about it, so we should be safe

Reply:
[This](https://mempool.space/tx/3e9d344993b210915c41e693df0bbf48af7a88f2f7011ad4e0264a3c69da8b33) is the original `TX_HEX` btw.